modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -3474,6 +3474,15 @@
 		],
 		"to": "getListType(CompanyThreadLocal.getCompanyId(), param#0#, param#1#)"
 	},
+	{
+		"classNames": [
+			"RoleLocalService",
+			"RoleLocalServiceUtil"
```

Comment:
@arodriguesrm did you check if the pattern isn't the same for the RoleService and RoleServiceUtil?